.github/workflows/pr.yml

```
@@ -0,0 +1,172 @@
+name: Config File Check
+description: Check config files are valid by performing a betaflight build (using master)
```

actionlint (1.7.12)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)